ArduCopter/heli.cpp

```diff
@@ -151,6 +151,18 @@ float Copter::get_pilot_desired_rotor_speed() const
     return 0.0f;
 }
 
+// get throttle from transmitter
+// returns -1 if throttle hasn't been defined
+float Copter::get_pilot_desired_throttle() const
+{
+    RC_Channel *rc_ptr = rc().find_channel_for_option(RC_Channel::AUX_FUNC::MOTOR_INTERLOCK);
+    if (rc_ptr != nullptr) {
+        rc_ptr->set_range(1000);
```

Comment:
It's a bit odd to have a set_range() here in a get method.  If possible I would move this to where we do other set_range calls.

Reply:
is the "RC_Channel::AUX_FUNC::MOTOR_INTERLOCK" part correct?

Reply:
> is the "RC_Channel::AUX_FUNC::MOTOR_INTERLOCK" part correct?

why do you think this is incorrect?   Truthfully I'm pretty sure I copied from another part of the code.  I'll check.